Bluetooth: Mesh: Point to point and Mesh DFU (#14833)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* :ref:`ble_mesh_dfu_target` sample:

  * Added a note on how to compile the sample with new Composition Data.

>   * Added point-to-point DFU support by adding :file:`overlay-ptp_dfu.conf` overlay.

Reviewer: ```suggestion
  * Added point-to-point DFU support with overlay file :file:`overlay-ptp_dfu.conf`.
```

>   * Added a note on how to compile the sample with new Composition Data.
>   * Added point-to-point DFU support with overlay file :file:`overlay-ptp_dfu.conf`.

Reviewer: ```suggestion
  * Added:
  
    * A note on how to compile the sample with new Composition Data.
    * Point-to-point DFU support with overlay file :file:`overlay-ptp_dfu.conf`.
```
